src/server/index.js

```diff
@@ -0,0 +1,54 @@
+import express from 'express';
+import React from 'react';
+import { renderToString } from 'react-dom/server';
+import App from '../shared/App';
+
+const app = express();
+
+app.use(express.static('public'));
+
+app.get('/public', (req, res) => {
+  res
+    .status(404) // HTTP status 404: NotFound
+    .send('Not found');
+});
+
+const isDev = process.env.NODE_ENV === 'development';
+
+let sriptPaths = [];
+const cssPath = isDev
```

Comment:
Let's avoid this distinguishes and keep it simple - use the same bundle files for both develop and production (however, keep them different) to avoid this paths hardcoding